package.json

```diff
@@ -35,7 +35,7 @@
     "serve-built-storybook": "npx http-server docs",
     "verify-node-version": "chmod +x ./scripts/verify-node-version.sh && ./scripts/verify-node-version.sh",
     "preversion": "node ./src/generateVersion.js",
-    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts && npm run compile",
+    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts && git add package.json",
```

Comment:
npm version command should handle adding package and package-lock itself. We shouldn't be manually adding those. They should be staged and included in the commit. Was it not adding these files when you tested?

Reply:
You're totally right. I removed this in the most recent commit.